packages/fast-tooling-react/src/form/templates/plugin.control.utilities.tsx

```diff
@@ -27,8 +54,25 @@ export default abstract class ControlPluginUtilities<
         this.updateConfig(config);
     }
 
+    /**
+     * Determines if there is a match for the IDs set for the plugin
+     * and a provided ID
+     */
+    public matchesId(id: string): boolean {
+        return this.config.id.indexOf(id) !== -1;
```

Comment:
why `indexOf` and not direct comparison? couldn't you get unexpected id matches?

Reply:
No, this is a array indexOf not string indexOf